gpt4all-chat/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+option(GPT4ALL_TRANSLATIONS OFF "Build with translations")
```

Comment:
why make this optional? I feel like its usually a space saving thing to allow disabling translations but they're going to be relatively tiny compared to everything else we bundle